src/main/java/uk/gov/hmcts/reform/civil/notify/NotificationsProperties.java

```diff
@@ -514,4 +514,22 @@ public class NotificationsProperties {
 
     @NotEmpty
     private String notifyApplicantLRMediationTemplate;
+
+    @NotEmpty
+    private String notifyLrClaimantSuccessfulMediation;
+
+    @NotEmpty
+    private String notifyOneVTwoClaimantSuccessfulMediation;
+
+    @NotEmpty
+    private String notifyLipClaimantSuccessfulMediation;
+
+    @NotEmpty
+    private String notifyLrDefendantSuccessfulMediation;
+
+    @NotEmpty
+    private String notifyTwoVOneDefendantSuccessfulMediation;
+
+    @NotEmpty
+    private String notifyLipDefendantSuccessfulMediation;
```

Comment:
lip have the same email, could we please use the same template please? 

these two can be combined:
    private String notifyLipClaimantSuccessfulMediation;
private String notifyLipDefendantSuccessfulMediation;

Reply:
fixed